bin/dbServer.py

```diff
@@ -31,12 +31,33 @@
 from flask import Flask, request
 import json
 import logging as log
+import os
 import sys
 
+import ConfigParser
+import sqlalchemy
+from sqlalchemy.engine.url import URL
+
 from lsst.dbserv import dbREST_v0
 
 app = Flask(__name__)
 
+def initEngine():
+    config = ConfigParser.ConfigParser()
+    defaults_file = os.path.expanduser('~/.lsst/dbAuth-testLocal.txt')
+    config.readfp(open(defaults_file))
+    db_config = dict(config.items("mysql"))
+    # Translate user name
+    db_config["username"] = db_config["user"]
+    del db_config["user"]
+    # SQLAlchemy part
+    url = URL("mysql",**db_config)
+    return sqlalchemy.create_engine(url)
+
+engine = initEngine()
+
+app.config["default_engine"] = engine
```

Comment:
The `engine` variable is not used, I'd assign result of `initEngine()` directly to `app.config`